guides/common/modules/ref_conditional-statements.adoc

```diff
@@ -3,12 +3,13 @@
 [id="Conditional_Statements_{context}"]
 = Conditional statements
 
+[role="_abstract"]
 In your templates, you might perform different actions depending on which value exists.
 To achieve this, you can use conditional statements in your ERB syntax.
 
 In the following example, the ERB syntax searches for a specific host name and returns an output depending on the value it finds:
 
-.Example input
+* Example input:
```

Comment:
```suggestion
* Example input:
+
```

IMO there's always a line with a plus sign between a step and code block. same applies to line 23.

Reply:
Applied.